website/docs/src/content/component-reference/storage-components/search-reference.mdx

```diff
@@ -0,0 +1,388 @@
+---
+title: Full-text & vector search reference
+description: Vector search components for semantic search and RAG workflows.
+---
+
+# Search Reference
+
+API reference for GenSX Cloud search components.
+
+## SearchProvider
+
+Provides vector search capabilities to its child components.
+
+### Import
+
+```tsx
+import { SearchProvider } from "@gensx/storage";
+```
+
+### Props
+
+| Prop | Type | Default | Description |
+|------|------|---------|-------------|
+| `defaultPrefix` | `string` | `undefined` | Optional prefix for all namespace keys |
+
+### Example
+
+```tsx
+import { SearchProvider } from "@gensx/storage";
+
+const Workflow = gensx.Component(
+  "SearchWorkflow",
+  ({ input }) => (
+    <SearchProvider>
+      <YourComponent input={input} />
+    </SearchProvider>
+  )
+);
+```
+
+## useSearch
```

Comment:
None of the reference docs include mentions of `useSearchStorage`, `useDatabaseStorage`, and `useBlobStorage` which allow you to the namespace operations within a workflow.

Reply:
I'm fine with leaving it our if you think that's best but wanted to mention it

Reply:
Ok. I went ahead and added Client and useStorage sections to each of the storage reference docs.